debian/.debhelper/generated/pop-plymouth-theme/installed-by-dh_install

```diff
@@ -0,0 +1 @@
+./pop-basic/
```

Comment:
`.debhelper` is also generated and shouldn't be committed. Less obviously, `debian/files` is also generated during build.

Reply:
removed too